scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,12 @@
+# set nvcc as the underlying C++ compiler for Open MPI
+export OMPI_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set nvcc as the underlying C++ compiler for MPICH
+export MPICH_CXX=$TRILINOS_DIR/packages/kokkos/config/nvcc_wrapper
+
+# set the original C++ compiler as the underlying compiler for the nvcc
+# wrapper
+export NVCC_WRAPPER_DEFAULT_COMPILER=$PATH_TO_CXX_COMPILER
+
+# set all cuda calls to blocking launch
+export CUDA_LAUNCH_BLOCKING=1
```

Comment:
@mhoemmen Mark, quick question: does using Tpetra require CUDA_LAUNCH_BLOCKING, or is it properly fenced and may work without it?